mofacts/client/views/home/profile.js

```diff
@@ -89,22 +93,54 @@ Template.profile.helpers({
 // Template Events
 
 Template.profile.events({
-
+  //Enable TDF
+  'click .enableTdf': function(event, instance) {
+      Meteor.call('toggleTdfPresence', event.target.tdfid, ENABLED, function(err, result) {
+        if (err) {
+          console.log(err);
+          alert('Error enabling TDF');
+        } else {
+          console.log('TDF enabled');
+          //update the enabledTdfs reactive var
+          const enabledTdfs = instance.enabledTdfs.get();
+          const tdfToEnable = enabledTdfs.find((tdf) => {
+            return tdf.tdfs.tutor.setspec._id === event.target.tdfid;
+          });
+          tdfToEnable.enabled = true;
+          instance.enabledTdfs.set(enabledTdfs);
+          //update the disabledTdfs reactive var
+          const disabledTdfs = instance.disabledTdfs.get();
+          const tdfToDisable = disabledTdfs.find((tdf) => {
+            return tdf.tdfs.tutor.setspec._id === event.target.tdfid;
+          });
+          tdfToDisable.enabled = false;
+          instance.disabledTdfs.set(disabledTdfs);
+        }
+      });
+  },
   //TDF Search
   'click #practiceTDFSearch': function(event, instance) {
+    $('#practiceTDFSearchResultsContainer').css('opacity', '1');
+  },
+  'keyup #practiceTDFSearch': function(event, instance) {
     const search = event.target.value;
     const enabledTdfs = instance.enabledTdfs.get();
-    const filteredTdfs = enabledTdfs.filter((tdf) => {
+    filteredTdfs = enabledTdfs.filter((tdf) => {
       return tdf.tdfs.tutor.setspec.lessonname.toLowerCase().includes(search.toLowerCase());
     });
+    //also search tags
+    filteredTdfs = filteredTdfs.concat(enabledTdfs.filter((tdf) => {
+      return tdf.tdfs.tutor.setspec.tags && tdf.tdfs.tutor.setspec.tags.toLowerCase().includes(search.toLowerCase());
```

Comment:
Searching for a TDF breaks here. 
`profile.js:133 Uncaught TypeError: tdf.tdfs.tutor.setspec.tags.toLowerCase is not a function`